Meadowlark-js/packages/meadowlark-authz-server/src/handler/RequestToken.ts

```diff
@@ -68,14 +68,14 @@ function parseRequestTokenBody(authorizationRequest: AuthorizationRequest): Pars
     try {
       unvalidatedBody = querystring.parse(authorizationRequest.body);
     } catch (error) {
-      Logger.debug(`${moduleName}.parseRequestTokenBody: Malformed body`, authorizationRequest.traceId);
+      Logger.debug(`${moduleName}.parseRequestTokenBody: Malformed body - ${error.message}`, authorizationRequest.traceId);
       return { isValid: false, failureMessage: 'Malformed body' };
```

Comment:
I was a little surprised that this was not updated. But I might be ok with that for the auth server.

Reply:
```suggestion
      return { isValid: false, failureMessage: `Malformed body: ${error.message}` };
```